sklearn/metrics/classification.py

```diff
@@ -186,6 +187,74 @@ def accuracy_score(y_true, y_pred, normalize=True, sample_weight=None):
     return _weighted_sum(score, sample_weight, normalize)
 
 
+def multilabel_confusion_matrix(y_true, y_pred):
+    """Compute True positive, False positive, True negative, False negative
+    for a multilabel classification problem.
```

Comment:
Please follow PEP 257: one liner short description, blank line, possibly multi-line detailed description.

Having a short first line is important for many dev tools (e.g. IDEs) and API documentation tools such as the [API summary page](http://scikit-learn.org/stable/modules/classes.html) built with sphinx.

```python
def multilabel_confusion_matrix(y_true, y_pred):
    """Compute the confusion matrix for a multilabel classification problem.

    Compute True positive, False positive, True negative, False negative rates for each
    binary classification sub-problem.

     ...
     """
     ...
```